src/algorithms.jl

```diff
@@ -449,8 +494,9 @@ A module listing the default values for the typical parameters in Krylov based a
   - `krylovdim = 30`: the maximal dimension of the Krylov subspace that will be constructed
   - `maxiter = 100`: the maximal number of outer iterations, i.e. the maximum number of
     times the Krylov subspace may be rebuilt
+  - `blockkrylovdim = 100`: the maximal dimension of the Krylov subspace that will be constructed for BlockLanczos
   - `tol = 1e-12`: the tolerance to which the problem must be solved, based on a suitable
-    error measure, e.g. the norm of some residual.
+    error measure, e.g. the norm of some residual. It's also used as the default value of `qr_tol`
```

Comment:
I would not add this extra sentence here; `qr_tol` is much more specific and might be meaningless to users that have not yet read about `BlockLanczos`.

```suggestion
    error measure, e.g. the norm of some residual. It's also used as the default value of `qr_tol`
```